CMakeLists.txt

```diff
@@ -55,12 +55,16 @@ add_library(thumbnailer_slope_optim_lib ${CMAKE_CURRENT_SOURCE_DIR}/src/thumbnai
 target_include_directories(thumbnailer_slope_optim_lib PUBLIC ${PROTOBUF_INCLUDE_DIRS})
 target_link_libraries(thumbnailer_slope_optim_lib protobuf::libprotobuf webp)
 
+add_library(thumbnailer_near_lossless_lib ${CMAKE_CURRENT_SOURCE_DIR}/src/thumbnailer_near_lossless.cc ${PROTO_SRCS})
+target_include_directories(thumbnailer_near_lossless_lib PUBLIC ${PROTOBUF_INCLUDE_DIRS})
+target_link_libraries(thumbnailer_near_lossless_lib protobuf::libprotobuf webp)
```

Comment:
```
add_library(thumbnailer_slope_optim_lib ${CMAKE_CURRENT_SOURCE_DIR}/src/thumbnailer_slope_optim.cc)

add_library(thumbnailer_near_lossless_lib ${CMAKE_CURRENT_SOURCE_DIR}/src/thumbnailer_near_lossless.cc )
```

Reply:
I do not think it is worth adding that many library targets. I think there should be only:

- thumbnailer_lib
- thumbnailer_utils_lib (optional and only if thumbnailer_lib can be built and used without the utils, to reduce binary size)
- thumbnailer binary
- compare binary

Reply:
We just have fixed that :)